app/pages/resource/resource-info/__tests__/ResourceInfo.test.js

```diff
@@ -66,6 +67,21 @@ describe('pages/resource/resource-info/ResourceInfo', () => {
     expect(reservationInfo.prop('resource')).toEqual(defaultProps.resource);
   });
 
+  test('renders specific terms heading', () => {
```

Comment:
You dont need to write those test, its non-sense, its ok not to write any this case